sal/class.json-api-site-base.php

```diff
@@ -149,6 +149,14 @@ public function woocommerce_is_active() {
 		return false;
 	}
 
+	public function is_cloud_eligible() {
+		return false;
+	}
+
+	public function get_products() {
+		return [];
+	}
```

Comment:
@jeherve - This should cascade to `class.json-api-site-jetpack.php`, correct?

Reply:
Yup, you're right! 👍